pkgs/development/python-modules/backgroundremover/default.nix

```diff
@@ -0,0 +1,77 @@
+{ buildPythonPackage
+, certifi
+, charset-normalizer
+, fetchPypi
+, ffmpeg-python
+, filelock
+, filetype
+, gdown
+, hsh
+, idna
+, lib
+, more-itertools
+, moviepy
+, numpy
+, pillow
+, pymatting
+, pysocks
+, requests
+, scikitimage
+, scipy
+, six
+, torch
+, torchvision
+, tqdm
+, urllib3
+, waitress
+}:
+
+buildPythonPackage rec {
+  pname = "backgroundremover";
+
+  version = "0.2.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-YAiPwI7YCTIDJxgzEoVCT/6htRemcVuvfj2lVS3dlks=";
+  };
+
+  propagatedBuildInputs = [
+    certifi
+    charset-normalizer
+    ffmpeg-python
+    filelock
+    filetype
+    gdown
+    hsh
+    idna
+    more-itertools
+    moviepy
+    numpy
+    pillow
+    pymatting
+    pysocks
+    requests
+    scikitimage
+    scipy
+    six
+    torch
+    torchvision
+    tqdm
+    urllib3
+    waitress
+  ];
+
+  doCheck = false;
+
+  meta = with lib; {
+    mainProgram = "backgroundremover";
+    description = "Command line tool to remove background from image and video, made by nadermx to power";
+    platforms = platforms.all;
```

Comment:
Is this necessary for Python packages?

Reply:
`platforms` I guess not, but I started from that PR I mentioned

Reply:
removed